packages/aws-cdk/lib/util/cdk-chalk.ts

```diff
@@ -0,0 +1,55 @@
+import * as chalk from 'chalk';
+
+// Create a new chalk instance with color support disabled if we're not in a TTY
```

Comment:
To further explain this. The way chalk works is by wrapping text in ANSI codes, which if you are receiving the raw text value of, will show up in your message on a notify. So if we detect we are not in a TTY (and thus likely being used programmatically, or at least in some shell env) we disable chalk